### PR TITLE
Crhodes/rpm spec

### PR DESCRIPTION
**Pull request by arreyder**
Adds a spec file suitable to building rpms for centos7.

### CODE REVIEW

File: dist/rpm/libxjwt.spec [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+rm -Rf $RPM_BUILD_DIR/libxjwt-%{version}
+tar xvfz $RPM_SOURCE_DIR/v%{version}.tar.gz -C $RPM_BUILD_DIR/
+
+%build
+cd libxjwt-%{version}
+scons build
+
+%install
+cd libxjwt-%{version}
+scons install --install-sandbox="$RPM_BUILD_ROOT"
```
Comment (pquerna) [inline dist/rpm/libxjwt.spec]: should we just add `--prefix=/usr` here, and then we can drop all of the `ld.so.conf.d` dance?  Seems like the most "normal" for an RPM to prefix in /usr?

File: dist/rpm/libxjwt.spec [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+%build
+cd %{name}-%{version}
+scons build
+
+%install
+cd %{name}-%{version}
+scons install --install-sandbox="$RPM_BUILD_ROOT" prefix=/usr
+
+%post -p /sbin/ldconfig
```
Comment (pquerna) [inline dist/rpm/libxjwt.spec]: can be removed?

Comment (arreyder) [reply dist/rpm/libxjwt.spec]: I think without this, scons will install to the root of the host and not the buildroot for the rpm.

Also, I dont think there's any other mechanism to have ldd pick up on the new module without that call there.  I can try removing both of these though and see what happens.

File: dist/rpm/libxjwt.spec [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+Summary:        Minimal C library for validation of real-world JWTs
+License:        Apache 2.0
+Group:          System/Libraries
+Source0:        https://github.com/ScaleFT/libxjwt/archive/v%{version}.tar.gz
+
+URL:            https://github.com/ScaleFT/libxjwt
+Vendor:         ScaleFT http://scaleft.com
+
+BuildRequires:  scons
```
Comment (pquerna) [inline dist/rpm/libxjwt.spec]: can be dropped

**Review by pquerna — APPROVED** [review]
LGTM

**Pull request merged**